HISTORY.md

```diff
@@ -1,12 +1,21 @@
 # cloudpathlib Changelog
 
-## 0.18.0 (2024-02-25)
+## v0.18.1 (2024-02-26)
+
+- Fixed import error due to incompatible google-cloud-strage by adding a version floor. ([Issue #408](https://github.com/drivendataorg/cloudpathlib/issues/408), (PR #410)[https://github.com/drivendataorg/cloudpathlib/pull/410]) 
```

Comment:
Thanks, this all looks good for our purposes. Minor nit: “by adding a version floor” is probably no longer an accurate description of this version of the fix.

Also typo:
```suggestion
- Fixed import error due to incompatible google-cloud-storage by adding a version floor. ([Issue #408](https://github.com/drivendataorg/cloudpathlib/issues/408), (PR #410)[https://github.com/drivendataorg/cloudpathlib/pull/410]) 
```